base/random.jl

```diff
@@ -1166,6 +1166,30 @@ immutable UUID
 end
 UUID(u::AbstractString) = convert(UUID, u)
 
+function uuid1(rng::AbstractRNG=GLOBAL_RNG)
+    u = rand(rng, Uint128)
+
+    # mask off clock sequence and node
+    u &= 0x00000000000000003fffffffffffffff
+
+    # set the unicast/multicast bit
+    u |= 0x00000000000000000000010000000000
+
+    # 0x01b21dd213814000 is the number of milliseconds between
+    # the UUID epoch and Unix epoch
+    timestamp = round(Uint64, time() * 1e7) + 0x01b21dd213814000
+    ts_low = timestamp & typemax(Uint32)
+    ts_mid = (timestamp >> 32) & typemax(Uint16)
+    ts_hi_and_ver = (timestamp >> 48) & 0x0fff
+    ts_hi_and_ver |= (1 << 12)
+
+    u |= convert(Uint128, ts_low) << 96
```

Comment:
In 0.4, the syntax is `UInt128(ts_low)` etc.   Note also that the capitalization has changed.